test/unit/tracks/html-track-element.test.js

```diff
@@ -27,23 +27,19 @@ QUnit.test('can create a html track element with various properties', function(a
   const kind = 'chapters';
   const label = 'English';
   const language = 'en';
-  const src = 'http://www.example.com';
```

Comment:
It would be nice but it ends up causing an XHR which logs an error in IE8 and there isn't a good way to cancelling the track.
Text Tracks have a source is already tested elsewhere. I think not having an extra error logged for a failed XHR is better than testing something that is partially tested elsewhere anyway.